MAUI/DataForm/DataForm-settings.md

```diff
@@ -0,0 +1,899 @@
+---
+layout: post
+title: Change the data editor settings in .NET MAUI DataForm control | Syncfusion
+description: Learn about to change the Syncfusion .NET MAUI DataForm (SfDataForm) settings in mobile and desktop applications from a single shared codebase.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Change the data editor settings (SfDataForm)
+
+## Auto generate data editor 
+
+By default, the `SfDataForm.Items` will be generated based on the property type. For example, the `DataFormTextItem` will be generated based on the property type in `DataObject`. 
+The `DataFormItem` generation depends on the type and attribute defined for the property.
+You can customize the auto generated editor, label and group settings by using the `GenerateDataFormItem` event.
+
+<table>
+<tr>
+<th>Generated DataFormItem Type</th>
+<th>Editor</th>
+<th>Data Type / Attribute</th>
+</tr>
+<tr>
+<td>
+DataFormTextItem
+</td>
+<td>
+{{'[Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for the string type and the properties with [DataType(DataType.Text)] attributes.
+</td>
+</tr>
+<tr>
+<td>
+`DataFormMultilineTextItem`
+</td>
+<td>
+{{'[Editor](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for the string type and the properties with  [DataType(DataType.MultilineText)] attributes. 
```

Comment:
Generated for the string type properties with [DataType(DataType.MultilineText)] attribute.